### DISCUSSION TITLE
Adding register for Hot water start to NIBE S1156 / S1256 / S1155 / S1255

### DISCUSSION THREAD

**Discussion by KptnHaddock** [yozik04/nibe#196] [General]
Hi,
Thank you for initiating and mainting this integration. I have now used it for some months and it works great with HA and my newly installed Nibe s1256.
Have been trying to understand why one particual register listed in the Nibe Modbus technical desciption does not show up in the integration, the Hot water start (BT5). My understanding is the s1256 integration is based on extension file nibe/data/s1155_s1255.json, where this particual register seems to be missing. Whould it be possible to somehow get it in?

Here is how Hot water start is defined in the .csv file, exported from the heatpump.

| Title | Register type | Register | Division factor | Unit | Size of variable | Min value| Max value | Default value |
|------------------|-----------------|------------------|-----------------|------------------|------------------|------------------|------------------|------------------|
| Hot water start (BT5) | MODBUS_INPUT_REGISTER | 2014 | 10 | Â°C | 2 | 0 | 0 | 0 |

I would be happy to contribute but need some guidance on what to edit where as my gihub experience is very limited.

**Comment by yozik04** [comment]
Please export full CSV and we will merge it in.

**Reply by KptnHaddock** [reply]
Sounds great!
please find a fresh exported .csv from Nibe s1256 here:
[modbus_addresses_all_20241227-06569924138015-1.csv](https://github.com/user-attachments/files/18261126/modbus_addresses_all_20241227-06569924138015-1.csv)

**Reply by yozik04** [reply]
Addressed in https://github.com/yozik04/nibe/pull/197
